packages/web-components/src/donut-chart/donut-chart.options.ts

```diff
@@ -0,0 +1,40 @@
+export interface IChartDataPoint {
```

Comment:
Others with a bit more historical background on our types can comment but we don't typically use interfaces like this, we typically just create a `type` without the `I*` prefix.

Reply:
Thanks for suggesting. We will remove the I prefix